static/js/upload_widget.ts

```diff
@@ -25,17 +39,92 @@ export function build_widget(
     $upload_button: JQuery,
     $preview_text?: JQuery,
     $preview_image?: JQuery,
+    $select_button?: JQuery,
```

Comment:
Why is this called `$select_button`?

Reply:
$select_button refers to the button that is used to select the cropped image, which is why I named it as such. However, I am open to suggestions for alternative names. If you have any other ideas for what to name the button, I would be happy to hear them and consider making the change.